cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -973,6 +973,86 @@ def submit_aces(
             lifespan=lifespan,
         )
 
+    def submit_cb(
+        self,
+        target: str,
+        shots: int,
+        circuits: cirq.Circuit,
+        num_channels: int,
+        num_sequences: int,
+        depths: list[int],
+        method: str | None = None,
+        noise: str | cirq.NoiseModel | None = None,
+        error_prob: float | tuple[float, float, float] | None = None,
+    ) -> str:
+        """Submits the jobs to characterize `target` through the Cycle Benchmarking protocol.
+
+        The protocol in detail can be found in: https://arxiv.org/abs/1902.08543.
+
+        Args:
+            target: The device target to characterize.
```

Comment:
```suggestion
            target: The target device to characterize.
```
(ditto in superstaq_client)